frontend/src/components/forms/order/helper.ts

```diff
@@ -0,0 +1,10 @@
+import {IProduct} from "../../../interfaces/IProduct";
+import {IOrderItem} from "../../../interfaces/IOrder";
+
+export function productsBySupplier (this: string, product:IProduct):boolean {
```

Comment:
By reading the name of this method I wouldn't expect it to return a boolean

Reply:
Yeah you're right, where I use it thoug it's written like this
```
const productOptions = mapProductsToSelectData(products.filter(productsBySupplier, selectedSupplierId))
```